server/src/main/java/dev/slimevr/vr/trackers/IMUTracker.java

```diff
@@ -490,40 +492,40 @@ public void resetMounting(boolean reverseYaw) {
 		mountRotFix.set(buffer);
 
 		// Get the difference from the last adjustment
-		buffer.multLocal(lastRotAdjust.inverseLocal());
+		buffer.multLocal(lastRotAdjust.inverse());
 		// Apply the yaw rotation difference to the yaw fix quaternion
-		yawFix.multLocal(buffer.inverseLocal());
+		yawFix.multLocal(buffer.inverse());
```

Comment:
It's buffer variable, do not make unnecessary copies, it should be inverseLocal()